src/data/multiset/locally_finite.lean

```diff
@@ -10,66 +10,150 @@ import data.finset.locally_finite
 
 This file provides basic results about all the `multiset.Ixx`, which are defined in
 `order.locally_finite`.
-
-## TODO
-
-Bring the lemmas about `multiset.Ico` in `data.multiset.intervals` here and in `data.nat.interval`.
 -/
 
 variables {α : Type*}
 
 namespace multiset
 section preorder
-variables [preorder α] [locally_finite_order α] {a b : α}
+variables [preorder α] [locally_finite_order α] {a b c : α}
 
 lemma nodup_Icc : (Icc a b).nodup := finset.nodup _
 
+lemma nodup_Ico : (Ico a b).nodup := finset.nodup _
+
 lemma nodup_Ioc : (Ioc a b).nodup := finset.nodup _
 
 lemma nodup_Ioo : (Ioo a b).nodup := finset.nodup _
 
 @[simp] lemma Icc_eq_zero_iff : Icc a b = 0 ↔ ¬a ≤ b :=
 by rw [Icc, finset.val_eq_zero, finset.Icc_eq_empty_iff]
 
+@[simp] lemma Ico_eq_zero_iff : Ico a b = 0 ↔ ¬a < b :=
+by rw [Ico, finset.val_eq_zero, finset.Ico_eq_empty_iff]
+
 @[simp] lemma Ioc_eq_zero_iff : Ioc a b = 0 ↔ ¬a < b :=
 by rw [Ioc, finset.val_eq_zero, finset.Ioc_eq_empty_iff]
 
 @[simp] lemma Ioo_eq_zero_iff [densely_ordered α] : Ioo a b = 0 ↔ ¬a < b :=
 by rw [Ioo, finset.val_eq_zero, finset.Ioo_eq_empty_iff]
 
 alias Icc_eq_zero_iff ↔ _ multiset.Icc_eq_zero
+alias Ico_eq_zero_iff ↔ _ multiset.Ico_eq_zero
 alias Ioc_eq_zero_iff ↔ _ multiset.Ioc_eq_zero
 
 @[simp] lemma Ioo_eq_zero (h : ¬a < b) : Ioo a b = 0 :=
 eq_zero_iff_forall_not_mem.2 $ λ x hx, h ((mem_Ioo.1 hx).1.trans (mem_Ioo.1 hx).2)
 
-@[simp] lemma Icc_eq_zero_of_lt (h : b < a) : Icc a b = 0 :=
-Icc_eq_zero h.not_le
+@[simp] lemma Icc_eq_zero_of_lt (h : b < a) : Icc a b = 0 := Icc_eq_zero h.not_le
 
-@[simp] lemma Ioc_eq_zero_of_le (h : b ≤ a) : Ioc a b = 0 :=
-Ioc_eq_zero h.not_lt
+@[simp] lemma Ico_eq_zero_of_le (h : b ≤ a) : Ico a b = 0 := Ico_eq_zero h.not_lt
 
-@[simp] lemma Ioo_eq_zero_of_le (h : b ≤ a) : Ioo a b = 0 :=
-Ioo_eq_zero h.not_lt
+@[simp] lemma Ioc_eq_zero_of_le (h : b ≤ a) : Ioc a b = 0 := Ioc_eq_zero h.not_lt
```

Comment:
This type of reformatting makes it quite hard to tell what actually changed (while otherwise being fine). Can you try to do this type of thing in a separate PR in future?

Reply:
Yeah, sorry... #10087